simulator/obj_canvas.py

```diff
@@ -154,36 +154,9 @@ def set_background_color(self, color):
     Args:
       color: The color to set it to. """
     self.__canvas.config(bg=color)
-  def draw_grid(self):
-    grid = []
-    id = self.__canvas.create_line(200,0,200,1000)
-    grid.append(id)
-    id = self.__canvas.create_line(400,0,400,1000)
-    grid.append(id)
-    id = self.__canvas.create_line(600,0,600,1000)
-    grid.append(id)
-    id = self.__canvas.create_line(800,0,800,1000)
-    grid.append(id)
-    id = self.__canvas.create_line(1000,0,1000,1000)
-    grid.append(id)
-    id = self.__canvas.create_line(1200,0,1200,1000)
-    grid.append(id)
-    id = self.__canvas.create_line(1400,0,1400,1000)
-    grid.append(id)
-    id = self.__canvas.create_line(0,200,1600,200)
-    grid.append(id)
-    id = self.__canvas.create_line(0,400,1600,400)
-    grid.append(id)
-    id = self.__canvas.create_line(0,600,1600,600)
-    grid.append(id)
-    id = self.__canvas.create_line(0,800,1600,800)
-    grid.append(id)
-    id = self.__canvas.create_line(0,1000,1600,1000)
-    grid.append(id)
-    return grid
-  def clear_grid(self, grid):
-    for i in grid:
-      self.__canvas.delete(i)
+  def add_line(self, x_1, y_1, x_2, y_2):
```

Comment:
Instead of just creating an ```add_line``` function, how about creating an entire ```Line``` class? Similar to ```Rectangle``` and such.